etc/git_prompt_current_env_name.sh

```diff
@@ -0,0 +1,7 @@
+kw_get_current_env_name()
+{
+  kw_env_file=$(git rev-parse --show-toplevel 2> /dev/null)/.kw/env.current
```

Comment:
I don't think we need `git rev-parse`. Why not use `$PWD`? Keep in mind that kw might work in a kernel without git.